test/org/opensolaris/opengrok/web/UtilTest.java

```diff
@@ -343,12 +358,19 @@ public void testLinkify() {
         assertTrue(Util.linkify("smtp://example.com/OpenGrok/OpenGrok").equals("smtp://example.com/OpenGrok/OpenGrok"));
         assertTrue(Util.linkify("just some crazy text").equals("just some crazy text"));
 
-        // escaping tests
+        // escaping url
+        assertTrue(Util.linkify("http://www.example.com/\"quotation\"/else")
+                .contains("href=\"" + Util.encodeURL("http://www.example.com/\"quotation\"/else") + "\""));
```

Comment:
These are tests only for the url in href. If I was to do the tests for the whole content the test would look quite complicated (because the encode has to be called three times).

Also this doesn't prevent from future updates of the `linkify` method to contain some additional parameters.